### PR TITLE
updates the home page layout

### PR DESCRIPTION
**Pull request by itisdb**
Updates to the home layout. Added cards for the FAQs. Added college details cards. Added section for the Psychometric test. Added the Social Buttons.

### CODE REVIEW

**Comment by chayan007** [comment]
@itisdb add the SS of the new change in the description. Lets keep a basic log of what PR was about (alongside description that already exists)

File: templates/pages/public/home.html [hunk 1]
```diff
@@ -39,32 +43,1001 @@ <h5 class="text-white no-margin">AI supported college suggester for you speciall
...
+                                    <option value="GF">French Guiana</option>
+                                    <option value="PF">French Polynesia</option>
+                                    <option value="TF">French Southern Territories</option>
+                                    <option value="GA">Gabon</option>
+                                    <option value="GM">Gambia</option>
+                                    <option value="GE">Georgia</option>
+                                    <option value="DE">Germany</option>
+                                    <option value="GH">Ghana</option>
+                                    <option value="GI">Gibraltar</option>
+                                    <option value="GR">Greece</option>
```
Comment (chayan007) [inline templates/pages/public/home.html:417]: Remove other countries than India. Best to keep state based form options (for India)

Comment (itisdb) [reply templates/pages/public/home.html:417]: Forms data will be updated. This is the basic layout.

**Comment by itisdb** [comment]
please do have a check I sent you the Snapshots

**Pull request merged**